content/docs/start/experiments.md

```diff
@@ -19,13 +19,38 @@ the [`example-dvc-experiments`][ede] project.
 
 [ede]: https://github.com/iterative/example-dvc-experiments
 
+## Initializing a project into DVC experiments
+
+If you already have a DVC project, that's great. You can start to use `dvc exp`
+commands right away to create experiments in your project. (See the [user's
+guide] for detailed information.) In this section, we'll focus how to structure
+an ML project into a DVC experiments project with `dvc exp init`.
+
+A typical machine learning project has some data, a set of scripts that trains a
+model, a bunch of hyperparameters that modify these models. DVC makes certain
+assumptions about the names of these elements to initialize an experimentation
+project with:
+
+```dvc
+$ dvc exp init python src/train.py
```

Comment:
Example that was being used for https://github.com/iterative/dvc.org/pull/2979 (it's since been cut down):

```
$ dvc exp init -i
This command will guide you to set up a default stage in dvc.yaml.
See https://dvc.org/doc/user-guide/project-structure/pipelines-files.

DVC assumes the following workspace structure:
├── data
├── metrics.json
├── models
├── params.yaml
├── plots
└── src

Command to execute: python src/train.py
Path to a code file/directory [src, n to omit]: src/train.py
Path to a data file/directory [data, n to omit]: data/images/
Path to a model file/directory [models, n to omit]:
Path to a parameters file [params.yaml, n to omit]:
Path to a metrics file [metrics.json, n to omit]:
Path to a plots file/directory [plots, n to omit]: logs.csv
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
default:
  cmd: python src/train.py
  deps:
  - data/images/
  - src/train.py
  params:
  - model
  - train
  outs:
  - models
  metrics:
  - metrics.json:
      cache: false
  plots:
  - logs.csv:
      cache: false
Do you want to add the above contents to dvc.yaml? [y/n]: y

Created default stage in dvc.yaml. To run, use "dvc exp run".
See https://dvc.org/doc/user-guide/experiment-management/running-experiments.
```

Reply:
I think `--interactive` deserves a mention, though the screen estate it occupies may be too large.